banners/mobile_english/styles/theme_overrides_ctrl.pcss

```
@@ -0,0 +1,3 @@
+.wmde-banner-navigation-wrapper {
+	padding-bottom: 57px !important;
+}
```

I figured out how to override the theme file without using `!important`

1. Use the same nesting rules as in the original files. Apparently, PostCSS puts the non-nested classes at the top and then adds the concatened classes from nested rules after, regardless of the order in the file.
2. Use the media query, it adds specificity and overrides previous rules

```suggestion
.wmde-banner {
   &-navigation-wrapper {
	padding-bottom: 57px;
	@media ( min-width: $breakpoint-phone-s ) {
			padding-bottom: 57px;
	}
   }
}
```